simularium/simularium-website
# Feature/processing overlay

Problem
=======
Next step in the autoconversion workflow is to have the processing overlay fill the screen, and a cancel modal to back out of it. Merging into autoconversion mega ticket branch 346.

Closes [#390](https://github.com/simularium/simularium-website/issues/390) and [#408](https://github.com/simularium/simularium-website/issues/408)



Solution
========
* Bug fix (non-breaking change which fixes an issue)
* New feature (non-breaking change which adds functionality)

Change summary:
---------------
1) Main body of work is adding the conversion overlay and cancel modal.
2) Small bug fix from previous PR that allowed multiple file uploads.

Steps to Verify:
----------------
1. Select 'Import other file type' from Load Model menu
2. Select engine and file to upload, click Next.
3. Check out styling of page, test Cancel and Open New Viewer buttons.
4. Use buttons on modal when cancelling
